src/partials/akeneo-event-platform.handlebars

```diff
@@ -0,0 +1,45 @@
+{{!-- Shared template for the documentation page and guide pages, as well as the PHP client doc --}}
+{{#> layout title=title description=description}}
+    <div class="container">
+        <div class="row">
+            <div class="col-md-10 col-xs-9">
+                <h1>
+                    {{!-- TO DO AKENEO EVENT PLATFORM LOGO --}}
+                    {{!-- <img src="../../img/akeneo-event-platform/logo.png"/> --}}
```

Comment:
@monikaakeneo do we have a clean logo ? we receive one on google drive but are they validated ?

Reply:
@AdelELHAIBA-Akeneo - DA created the logo, so they validated it. We can use it for now, and I will double-check with Sarah if she has any objections.